### ISSUE TITLE
Messaging extensions without the parameters object cannot be configured

### ISSUE THREAD

**Issue by wictorwilen** [MicrosoftDocs/msteams-docs#648]
When building a Teams App with a messaging extension that doesn't require the `parameters` object (when using task modules or adaptive cards for instance) you get a script error when clicking on the messaging extension ellipsis (...)

```
TypeError: Cannot read property 'forEach' of undefined
    at t.runCommand (https://statics.teams.microsoft.com/hashedjs/3.2-app.min-52be5740.js:2:15360)
    at e.onContextMenuShowing (https://statics.teams.microsoft.com/hashedjs/3.2-app.min-52be5740.js:4:906024)
    at Object.a.onBeforeShow (https://statics.teams.microsoft.com/hashedjs/3.1-app.min-c5943dfa.js:1962:1772)
    at Object.T.show (https://statics.teams.microsoft.com/hashedjs/1-vendor.min-9ed9623a.js:9:5653)
    at Object.T.enter (https://statics.teams.microsoft.com/hashedjs/1-vendor.min-9ed9623a.js:9:5485)
    at Object.T.toggle (https://statics.teams.microsoft.com/hashedjs/1-vendor.min-9ed9623a.js:9:6830)
    at https://statics.teams.microsoft.com/hashedjs/3.2-app.min-52be5740.js:4:902068
    at https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:155:246
    at e (https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:41:350)
    at https://statics.teams.microsoft.com/hashedjs/0-angular-jquery.min-8200dd6f.js:44:275 undefined
```

Excerpt from the manifest:
``` json
{
          "id": "adaptiveAdaptiveMessageExtension",
          "title": "adaptive-adaptive",
          "description": "Add a clever description here",
          "initialRun": true,
          "type": "action",
          "context": [
            "compose",
            "commandBox",
            "message"
          ],
          "fetchTask": true
        },
```

Just adding a "dummy" `parameters` object solves it

**Comment by Gousia-MSFT** [comment]
@wictorwilen Could you please try using [manifest version](https://developer.microsoft.com/en-us/json-schemas/teams/v1.4/MicrosoftTeams.schema.json) 1.4?

**Comment by wictorwilen** [comment]
Thanks, that works as a workaround, but in 1.4 `context` for instance is not supported and a lot of other messaging extensions does not work

**Comment by Gousia-MSFT** [comment]
@wictorwilen Could you please help us understand what other features in message extension you are looking at and does not work in 1.4?

**Comment by wictorwilen** [comment]
@Gousia-MSFT - as mentioned above. `context` is not available in 1.4 and default values for parameters.

**Comment by billbliss** [comment]
I think it's a bug. We'll create a work item to fix it.

**Comment by wictorwilen** [comment]
Agreed @billbliss. Thanks

**Comment by the-Dust** [comment]
Hi! Can anybody tell, is it possible to create action messaging extension without any parameters and thus without need to submit it? I just want to click on my app in the "More actions" menu like this 
![messageextensions_messageaction 1](https://user-images.githubusercontent.com/20787804/59105294-bb9e3580-8944-11e9-8702-0b80d54aa102.png)
 and then see my chat again without any modal windows (while the activity is sending to my bot). And then I want to return a ComposeExtensionAttachment and insert it into a message.
Any help appreciated

**Comment by Wajeed-msft** [comment]
@wictorwilen - We have fixed this issue in [latest manifest schema 1.5](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema).
